tests/test_modes.py

```diff
@@ -3,14 +3,15 @@
 import pandas as pd
 import pandas.testing as pd_test
 import pytest
-from accern_data.accern_data import (
+
+from packages.python.accern_data.accern_data import (
     create_data_client,
     CSVMode,
     JSONMode,
     Mode,
     ModeType,
 )
-from accern_data.util import load_json
+from packages.python.accern_data.util import load_json
```

Comment:
hmm, what is that?

Reply:
`accern_data` is installed package. This will give coverage for test file only because it(coverage program) won't go inside the package.
`packages.python.accern_data` local module. We get better coverage this way because it can access the files.